packages/react-native-sdk/src/components/LocalVideoView.tsx

```diff
@@ -56,29 +63,59 @@ export const LocalVideoView = (props: LocalVideoViewProps) => {
   const isCameraOnFrontFacingMode = useStreamVideoStoreValue(
     (store) => store.isCameraOnFrontFacingMode,
   );
+  const pan = useRef(new Animated.ValueXY()).current;
+  const panResponder = useRef(
+    PanResponder.create({
+      onMoveShouldSetPanResponder: () => true,
+      onPanResponderMove: Animated.event([null, { dx: pan.x, dy: pan.y }], {
+        useNativeDriver: false,
+      }),
+      onPanResponderRelease: () => {
+        pan.extractOffset();
```

Comment:
what does this do? 

btw with the release, we can snap to positions... Can this be added in this PR?

Reply:
> Limitations: no snapping to top right/left, bottom right/left corners.

Agree, and I mentioned the limitation indeed, going to timebox the snapping feat.

Reply:
This will take to much time to figure out atm, going to park this ;)